addon-test-support/@ember/test-helpers/-utils.ts

```diff
@@ -9,7 +9,7 @@ export const futureTick = setTimeout;
  @private
  @returns {Promise<void>} Promise which can not be forced to be ran synchronously
 */
-export function nextTickPromise() {
+export function nextTickPromise(): RSVP.Promise<void> {
```

Comment:
Hmm, this seems odd, why can’t it be `RSVPPromise<void>`?

Reply:
Also, I’d prefer to change this to native Promise...

Reply:
> why can’t it be `RSVPPromise<void>`?

I could, but I just rebased and now everywhere else we also use `RSVP.Promise`

> I’d prefer to change this to native Promise...

me too, but iirc somewhere we use `.finally()`, which means we must use RSVP here until we refactor that away

Reply:
Ok, I’m happy to work on removing that finally after this lands. At least we have some TS flow analysis support to track down where that happens now....